modules/ossm-control-plane-deploy-1x.adoc

```diff
@@ -111,8 +111,8 @@ $ oc get smcp -n istio-system
 The installation has finished successfully when the READY column is true.
```

Comment:
... when the STATUS column shows `InstallSuccessful` and the numbers in the READY column match.